kairos-cli/bin/error.rs

```diff
@@ -0,0 +1,21 @@
+use hex::FromHexError;
+use thiserror::Error;
+
+use crate::crypto::error::CryptoError;
+
+#[derive(Error, Debug)]
+pub enum CliError {
+    /// Cryptography error.
+    #[error("cryptography error: {error}")]
+    CryptoError {
+        #[from]
+        error: CryptoError,
+    },
+    // TODO: Add error for "Failed to parse hex string: {}"
```

Comment:
can this todo be removed?

Reply:
Good catch! Outdated TODO removed in https://github.com/cspr-rad/kairos/pull/17/commits/67be402a1cdc9dd9f3eb60875e06d1a8d00322d8.